ext/node/resolution.rs

```diff
@@ -340,6 +340,29 @@ impl NodeResolver {
     let package_subpath = package_subpath
       .map(|s| format!("./{s}"))
       .unwrap_or_else(|| ".".to_string());
+
+    let maybe_resolved_path2 = self.resolve_package_subpath(
+      &package_json,
+      &package_subpath,
+      referrer,
+      node_module_kind,
+      DEFAULT_CONDITIONS,
+      mode,
+      permissions,
+    );
+
+    if !mode.is_types() {
+      maybe_resolved_path2.with_context(|| {
+        format!(
+          "Failed resolving package subpath '{}' for '{}'",
+          package_subpath,
+          package_json.path.display()
+        )
+      })?;
+    } else if maybe_resolved_path2.is_err() {
+      maybe_resolved_path2?;
+    }
+
```

Comment:
I think we can remove this code and just do what's mentioned in the comment on line 384? My guess is the current `.with_context(|| "".to_string())` causes a test failure.